models/public/colorization-v2-norebal/colorization-v2-norebal.md

```diff
@@ -0,0 +1,111 @@
+# colorization-v2-norebal
+
+## Use Case and High-Level Description
+
+The `colorization-v2-norebal` model is one of the [colorization](https://arxiv.org/pdf/1603.08511)
+group of models designed to perform image colorization. For details
+about this family of models, check out the [repository](https://github.com/richzhang/colorization).
+
+The gray-scale or BGR image should be preprocessed first:
+1) divide image's values by 255
+2) convert the image to LAB
+3) extract l component with 50.0 subtracted from each value
+
+The model input is a blob that consists of a L-channel of image of 1x1x224x224.
+The model output is a blob that consists of AB channels of image of 1x2x56x56.
```

Comment:
I think that given that the outputs differ between the original and converted models, you shouldn't describe the output here to avoid confusion (and might as well remove the input too).

Reply:
@IRDonch corrected